firmware/fsm.c

```diff
@@ -1108,46 +1108,51 @@ void fsm_msgSetU2FCounter(SetU2FCounter *msg)
 void fsm_msgDebugLinkGetState(DebugLinkGetState *msg)
 {
 	(void)msg;
-	RESP_INIT(DebugLinkState);
 
-	resp->has_layout = true;
-	resp->layout.size = OLED_BUFSIZE;
-	memcpy(resp->layout.bytes, oledGetBuffer(), OLED_BUFSIZE);
+	// Do not use RESP_INIT because it clears msg_resp, but another message
+	// might be being handled
+	DebugLinkState resp;
+	_Static_assert(MSG_DEBUG_OUT_SIZE >= sizeof(resp), "DebugLinkGetState is too large");
```

Comment:
This line should be removed.  It is for checking that the msg_resp buffer is large enough, but it isn't used here.